### PR TITLE
Add Tooltip system based on Tooltippery

### PR DESCRIPTION
**Pull request by Banane9**
Based on [Psychpsyo/Tooltippery](https://github.com/Psychpsyo/Tooltippery). The only factor for backwards compatibility with old tooltippery content is supporting tooltips on buttons with a `Comment` component with the Text `TooltipperyLabel:`.

### CODE REVIEW

**Comment by Banane9** [comment]
Basic version of tooltips, huzzah.

The locale strings should be defined in a separate package, so that they can be updated without a whole release of the gamepack.
It may also need extra code for tooltips where more particular details are required to be added, or to properly determine what to show - e.g. Setting session roles only has their index as an argument.

Mods can also add tooltips to their buttons using the `LocalActionButtonExtensions.WithTooltip` extension methods.

File: MonkeyLoader.Resonite.Integration/UI/Tooltips/Tooltip.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            TextRenderer.VerticalAlign.Value = TextVerticalAlignment.Top;
+            TextRenderer.HorizontalAlign.Value = TextHorizontalAlignment.Left;
+            TextRenderer.Size.Value = 200 * Scale;
+            TextRenderer.Bounded.Value = true;
+            TextRenderer.BoundsSize.Value = new float2(700 * Scale, 1);
+            TextRenderer.BoundsAlignment.Value = Alignment.TopLeft;
+            TextRenderer.Color.Value = TooltipConfig.Instance.TextColor;
+
+            // back panel slot
+            var backPanelOffset = Root.AddLocalSlot("bgOffset");
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/UI/Tooltips/Tooltip.cs:67]: Should probably use the non-local config here

File: MonkeyLoader.Resonite.Integration/UI/Tooltips/Tooltip.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            backPanelOffset.LocalPosition = new float3(0, 0, 1);
+            var backPanel = backPanelOffset.AddLocalSlot("Background");
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/UI/Tooltips/Tooltip.cs:69]: and here

**Comment by Nytra** [comment]
Maybe the tooltips should only show up if you have the dev tool equipped?

**Review by Nytra — APPROVED** [review]
Looks good and I tested it

**Comment by Banane9** [comment]
> Maybe the tooltips should only show up if you have the dev tool equipped?

Well, they can appear on any kind of button, I just happened to pick some inspector ones to show it.

Also whoops on the other local slot ones lol

**Pull request merged**